server/services/personService.test.ts

```diff
@@ -52,6 +56,122 @@ describe('PersonService', () => {
     service = new PersonService(hmppsAuthClientBuilder, prisonApiClientBuilder, prisonerSearchClientBuilder)
   })
 
+  describe('getOffenceHistory', () => {
+    it('returns the index offence and sorted additional offences for a given booking with additional details about each offence', async () => {
+      const indexOffence = offenceHistoryDetailFactory.build({
+        mostSerious: true,
+        offenceCode: 'O2',
+        offenceDate: '2023-02-02',
+      })
+      const offenceForIndexOffence = offenceDtoFactory.build({ code: 'O2' })
+
+      const additionalOffenceOne = offenceHistoryDetailFactory.build({ offenceCode: 'O1', offenceDate: '2023-01-01' })
```

Comment:
I wonder if the factory should just use `faker.datatypes.boolean()` rather than `false`, then it gets set explicitly here, given we've tended not to assume defaults in the factories in the past?